aten/src/ATen/native/native_functions.yaml

```diff
@@ -283,17 +288,44 @@
 - func: imag(Tensor(a) self) -> Tensor(a)
   variants: function
 
+- func: _conj(Tensor(a) self) -> Tensor(a)
+  variants: function, method
+  dispatch:
+    CompositeExplicitAutograd: _conj
+
 - func: conj(Tensor(a) self) -> Tensor(a)
   variants: function, method
+  manual_cpp_binding: True
 
-- func: conj.out(Tensor self, *, Tensor(a!) out) -> Tensor(a!)
+- func: _conj_physical(Tensor self) -> Tensor
+  variants: function, method
   dispatch:
-    CPU, CUDA: conj_out
+    CompositeExplicitAutograd: _conj_physical
 
-- func: _conj(Tensor self) -> Tensor
-  variants: function
+- func: conj_physical(Tensor self) -> Tensor
+  variants: function, method
+
+- func: conj_physical.out(Tensor self, *, Tensor(a!) out) -> Tensor(a!)
   dispatch:
-    CompositeExplicitAutograd: _conj
+    CPU, CUDA: conj_physical_out
+
+- func: conj_physical_(Tensor(a!) self) -> Tensor(a!)
+  variants: function, method
+  dispatch:
+    CompositeExplicitAutograd: conj_physical_
+
+- func: _resolve_conj(Tensor(a) self) -> Tensor(a)
+  variants: function, method
+  dispatch:
+    CompositeExplicitAutograd: _resolve_conj
+
+- func: resolve_conj(Tensor(a) self) -> Tensor(a)
+  variants: function, method
+
+- func: resolve_conj_(Tensor(a!) self) -> Tensor(a!)
```

Comment:
As pointed out by @ngimel, we shouldn't add `resolve_conj_` since it flips the conj bit, which will cause incorrect behavior for the views on the input tensor.